src/scripts/main.js

```diff
@@ -1,7 +1,35 @@
 'use strict';
 
+function hideDiv(div) {
+  div.style.display = 'none';
+}
+
 const pushNotification = (posTop, posRight, title, description, type) => {
   // write code here
+  const mess = document.createElement('div');
+
+  mess.setAttribute('class', `notification ${type}`);
+
+  // mess.style.position = 'absolute';
```

Comment:
The line setting the position style is commented out. Uncomment `mess.style.position = 'absolute';` to ensure the notification is positioned correctly on the page.